SLHCUpgradeSimulations/Configuration/python/HCalCustoms.py

```diff
@@ -93,7 +93,13 @@ def customise_Hcal2017(process):
         process=customise_mixing(process)
     if hasattr(process,'simHcalTriggerPrimitiveDigis'):
         process.simHcalTriggerPrimitiveDigis.upgradeHF = cms.bool(True)
-    
+    if hasattr(process,'dqmoffline_step'):
```

Comment:
probably more reliable just to check hasattr(process,'digiTask' ) but I presume this is all going away in favor of eras soon...